website/client/src/components/tasks/spells.vue

```diff
@@ -34,34 +26,55 @@
       >
         <div slot="drawer-slider">
           <div class="container spell-container">
-            <div class="row">
+            <div class="spell-container-row">
               <!-- eslint-disable vue/no-use-v-if-with-v-for -->
               <div
                 v-for="(skill, key) in spells[user.stats.class]"
                 v-if="user.stats.lvl >= skill.lvl"
                 :key="key"
-                v-b-popover.hover.auto="skillNotes(skill)"
-                class="col-12 col-md-3"
+                :id="`spell_${skill.key}`"
                 @click="!spellDisabled(key) ? castStart(skill) : null"
               >
+                  <b-popover
+                    :target="`spell_${skill.key}`"
+                    triggers="hover"
+                    placement="top"
+                    custom-class="popover-class"
+                  >
+                    <div class="popover-wrapper">
+                      <div class="popover-title">
+                        <div class="popover-title-text">{{skill.text()}}</div>
+                        <div class="popover-mana">
+                          <div class="popover-svg-icon" v-html="icons.mana"></div>
```

Comment:
this icon should be a bit bigger according to zeplin (16x16, now it's 12x12) and should be aligned to the mana number
<img width="342" alt="Schermata 2020-11-28 alle 19 44 42" src="https://user-images.githubusercontent.com/589911/100523638-42d30f00-31b2-11eb-9100-9e4232f85b26.png">
<img width="407" alt="Schermata 2020-11-28 alle 19 45 41" src="https://user-images.githubusercontent.com/589911/100523653-4feffe00-31b2-11eb-8855-fa710ff7c7e4.png">